src/pages/security/components/_hero_component.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { BackgroundImage, Header } from 'components/elements'
+import { Container } from 'components/containers'
+import device from 'themes/device'
+import { localize } from 'components/localization'
+
+const StyledContainer = styled(Container)`
+    flex-direction: column;
+    height: 100%;
+    width: 60%;
+
+    @media ${device.tablet} {
+        padding: 0 2rem;
+        align-items: flex-start;
+        width: 100%;
+    }
```

Comment:
you can add the padding here for responsive. And normally we use `tabletL` for breakpoint, as `Show` component's default breakpoint is `tabletL`
```suggestion
    @media ${device.tablet} {
        padding: 80px 16px;
        align-items: flex-start;
        width: 100%;
    }
```